packages/dataviews/src/dataform-controls/input.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import { privateApis } from '@wordpress/components';
+import { useCallback, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import type { DataFormInputControlProps } from '../types';
+import { unlock } from '../lock-unlock';
+
+const { ValidatedTextControl } = unlock( privateApis );
+
+export default function Input< Item >( {
```

Comment:
I don't really love the name `Input` here, but couldn't think of something better. Any suggestions?

Reply:
I don't mind much any name, given this is never going to be exported anywhere. Perhaps go with `ValidatedText` or something along those lines?

A small nit: can this utility live under `dataform-controls/UTILS/input.tsx`? I don't want people to think this is actually a DataForm control.